dotnetv3/KMS/README.md

```diff
@@ -1,59 +1,47 @@
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
-The following AWS KMS code examples use the AWS SDK for .NET version 3.5 or later:
+## Code examples
 
-- [CreateAliasExample](CreateAliasExample/) - Create a new alias for an AWS KMS key.
-- [CreateGrantExample](CreateGrantExample/) - Create a new AWS KMS grant.
-- [CreateKeyExample](CreateKeyExample/) - Create a new AWS KMS key.
-- [DescribeKeyExample](DescribeKeyExample/) - Describe an AWS KMS key.
-- [DisableKeyExample](DisableKeyExample/) - Disable an AWS KMS key.
-- [EnableKeyExample](EnableKeyExample/) - Enable an AWS KMS key.
-- [ListAliasesExample](ListAliasesExample/) - List the AWS KMS aliases.
-- [ListGrantsExample](ListGrantsExample/) - List AWS KMS grants.
-- [ListKeysExample](ListKeysExample/) - List AWS KMS keys.
+### Single actions
+Code excerpts that show you how to call individual service functions.
 
-## ⚠️ Important
+- [Create an alias for a key](CreateAliasExample/CreateAlias.cs) (`CreateAliasAsync`)
+- [Create a grant for a key](CreateGrantExample/CreateGrant.cs) (`CreateGrantAsync`)
+- [Create a key](CreateKeyExample/CreateKey.cs) (`CreateKeyAsync`)
+- [Describe a key](DescribeKeyExample/DescribeKey.cs) (`DescribeKeyAsync`)
+- [Disable a key](DisableKeyExample/DisableKey.cs) (`DisableKeyAsync`)
+- [Enable a key](EnableKeyExample/EnableKey.cs) (`EnableKeyAsync`)
+- [List aliases for a key](ListAliasesExample/ListAliases.cs) (`ListAliasesAsync`)
+- [List grants for a key](ListGrantsExample/ListGrants.cs) (`ListGrantsAsync`)
+- [List keys](ListKeysExample/ListKeys.cs) (`ListKeysAsync`)
 
-- We recommend that you grant your code least privilege, or at most the minimum
-  permissions required to perform the task. For more information, see
-  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
-  in the AWS Identity and Access Management User Guide. 
-- This code has not been tested in all AWS Regions. Some AWS services are
-  available only in specific [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/).
-- Running this code might result in charges to your AWS account. 
-- Running the unit tests might result in charges to your AWS account. [optional]
+## Run the examples
 
-## Running the examples
+### Prerequisites
+* To find prerequisites for running these examples, see the
+  [README](../README.md#Prerequisites) in the dotnetv3 folder.
 
 The examples in this folder use the default user account. The call to
-initialize the AWS KMS client does not specify the AWS Region. Supply
+initialize the Kinesis client does not specify the AWS Region. Supply
```

Comment:
Was "Kinesis" intended here?

Reply:
Yes! And the sentence was fixed when I used the common client initialization paragraph.